nameserver/zone.go

```diff
@@ -3,151 +3,761 @@ package nameserver
 import (
 	"bytes"
 	"fmt"
+	"github.com/benbjohnson/clock"
 	"github.com/miekg/dns"
 	. "github.com/weaveworks/weave/common"
 	"net"
 	"sync"
+	"time"
 )
 
 const (
 	RDNSDomain = "in-addr.arpa."
 )
 
+const (
+	DefaultLocalDomain     = "weave.local." // The default name used for the local domain
+	DefaultRefreshInterval = int(localTTL)  // Period for background updates with mDNS
+	DefaultRelevantTime    = 60             // When to forget info about remote info if nobody asks...
+
+	defaultNumUpdaters    = 1              // Number of background updaters
```

Comment:
Perhaps this should default to something higher? `InsistentLookupName` is blocking...